tests/Functional/Controller/TruncateControllerTest.php

```diff
@@ -21,6 +21,9 @@
 
 final class TruncateControllerTest extends TestCase
 {
+    /**
+     * @requires extension gd_info
```

Comment:
IMO the extension is `gd`:
```suggestion
     * @requires extension gd
```